domain/model/astronaut.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+// Astronaut is a type
```

Comment:
As a general recommendation, try to be more explicit when commenting on an exported type, var, etc.